docs/lib/storage/fragments/js/copy.md

```diff
@@ -0,0 +1,116 @@
+## Copy
+
+`Storage.copy` method copies an existing file to a different key.
+
+It returns a `{key: new S3 Object key}` object on success:
+
+<amplify-callout>
+Storage.copy can copy an object up to 5 GB in a single operation.
+</amplify-callout
+
+### Options
+
+```typescript
+await Storage.copy(
+	src: { 
+		key: string,
+		level?: 'private' | 'protected' | 'public',
+		identityId?: string, // only works if you are copying from a protected file 
+	},
+	dest: { 
+		key: string,
+		level?: 'private' | 'protected' | 'public',
+	}, 
+	config?: {
+		track?: boolean, // if true, automatically send Storage Events to Amazon Pinpoint
+	}
+)
+```
+
+### Other Options
```

Comment:
remove this and say: 
"In addition, you can configure advanced capabilities such as cache control, meta data, and more."